mfhdf/hrepack/hrepacktst.c

```diff
@@ -999,21 +396,16 @@ add_gr_ffile(const char *name_file, int32 gr_id, const char *gr_name, int32 inte
  *
  * Return: SUCCEED, FAIL
  *
- * Programmer: Pedro Vicente, pvn@ncsa.uiuc.edu
- *
- * Date: August 18, 2003
- *
  *-------------------------------------------------------------------------
  */
-
 static int
 add_gr(const char  *gr_name,     /* gr name */
        int32        file_id,     /* file ID */
        int32        gr_id,       /* GR ID */
        int32        vgroup_id,   /* group ID */
        int32        chunk_flags, /* chunk flags */
        comp_coder_t comp_type,   /* compression flag */
-       comp_info   *comp_info /* compression structure */)
+       comp_info   *c_info /* compression structure */)
```

Comment:
The comp_info name conflicts with the type in hcomp.h. This is like saying `int *int`.